ts/modules/broken-axis.src.ts

```diff
@@ -19,33 +30,7 @@ import H from '../parts/Globals.js';
 declare global {
     namespace Highcharts {
         interface Axis {
-            breakArray?: Array<AxisBreakObject>;
-            unitLength?: number;
-            /** @requires modules/broken-axis */
-            isInAnyBreak(
-                val: (number|null|undefined),
-                testKeep?: boolean
-            ): (boolean|undefined);
-            /** @requires modules/broken-axis */
-            isInBreak(
-                brk: XAxisBreaksOptions,
-                val: (number|null|undefined)
-            ): (boolean|undefined);
-            /** @requires modules/broken-axis */
-            setBreaks(
-                breaks?: Array<XAxisBreaksOptions>,
-                redraw?: boolean
-            ): void;
-        }
-        interface AxisBreakBorderObject {
-            move: string;
-            size?: number;
-            value: number;
-        }
-        interface AxisBreakObject {
-            from: number;
-            len: number;
-            to: number;
+            breakArray: Array<AxisBreakObject>;
```

Comment:
Shouldn't this be moved to the `brokenAxis` object to keep the `Axis` clean from _broken_ stuff?

Reply:
Even though I had changed the demos, we have to be backward compatible here.